keras_cv/bounding_box/converters_text.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 The KerasCV Authors
```

Comment:
Shouldn't this file rather be named `converters_test.py` instead of `converters_text.py`?

Reply:
Wow good catch, thanks